code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2167,3 +2169,19 @@
 	M.adjustOrganLoss(ORGAN_SLOT_HEART, 0.25*REM)
 	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, 0.25*REM)
 	..()
+
+/datum/reagent/baldium
+	name = "Baldium"
+	description = "A major cause of hair loss across the world."
+	reagent_state = LIQUID
+	color = "#ecb2cf"
+	taste_description = "bitterness"
+
+/datum/reagent/baldium/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
```

Comment:
Use descriptive and obvious names for your variables. This goes for all the new code you add in your PR(s), you don't have to update old code (e.g. the ``to_chat`` lines above this)